.azure-pipelines.yml

```diff
@@ -269,6 +269,53 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt-get install -y python python-pip xvfb xorg
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py sdist bdist_wheel
+  - bash: |
+      set -eux
+      pip install --user virtualenv
+      python -m venv venv
+      source venv/bin/activate
+      pip install dist/*.whl[all]
+      xvfb run parsec core gui
+      deactivate
```

Comment:
Is this really needed ? (I genuinely don't know ^^)
I would guess using a different shell reset the env variables (hence reset the virtual env config) and the working directory

Reply:
Yup I'll remove those lines!